tests/README.md

```diff
@@ -126,29 +274,31 @@ correct, group structure is currently not used.
 Scenarios are currently supported only in `bash`. And type of scenario is
 defined by its file name.
 
-#### (something).pass.sh
+##### (something).pass.sh
 
 Success scenario - script is expected to prepare machine in such way that the
 rule is expected to pass.
 
-#### (something).fail.sh
+##### (something).fail.sh
 
 Fail scenario - script is expected to break machine so the rule fails. Test
-Suite than, if initial scan successfully fails, tries to remediate machine and
-expects success.
+Suite than, if initial scan fails as expected, tries to remediate machine and
+expects evaluation to pass.
 
-#### (something).sh
+##### (something).sh
 
-Support files, that are available to the scenarios during their runtime. Can
+Support files, these are available to the scenarios during their runtime. Can
 be used as common libraries.
 
+## How to write new test scenarios
+
 ### Scenarios format
 
 Scenarios are simple bash scripts. The scenarios start with a header which provides metadata.
 The header consists of comments (starting by `#`). Possible values are:
 - `platform` is a comma-separated list of platforms where the test scenario can be run. This is similar to `platform` used in our remediations. Examples of values: `multi_platform_rhel`, `Red Hat Enterprise Linux 7`, `multi_platform_all`. If `platform` is not specified in the header, `multi_platform_all` is assumed.
 - `profiles` is a comma-separated list of profiles which are using this scenario.
```

Comment:
I would update this sentence to "`profiles` is a comma-separated list of profiles to which this scenario applies to."